### DISCUSSION TITLE
Simple Python Interface

### DISCUSSION THREAD

**Discussion by MohamedAliRashad** [coqui-ai/TTS#469] [General]
I took a while to know how to use the package in my python code to generate voice from text (i later found `synthesize.py` and knew what i need to do).
Shouldn't be there a simpler interface with documentation for it in the README?

**Comment by Sadam1195** [comment]
Interface is also provided. Please read https://github.com/coqui-ai/TTS#example-synthesizing-speech-on-terminal-using-the-released-models for more details.

**Reply by stefan-reich** [reply]
Hi Sadam, the link is sadly not working anymore.

However, running server.py seems easy enough. I just used "make system-deps" and then "python3 TTS/server/server.py".

**Reply by Sadam1195** [reply]
yeah, many things have been changed since that comment posted. With latest release you can inference with instructions listed here in docs. https://tts.readthedocs.io/en/latest/inference.html
@stefan-reich
